src/coreclr/src/vm/interpreter.cpp

```diff
@@ -8698,7 +8698,7 @@ void Interpreter::UnboxAny()
                     CorElementType type1 = pMT1->GetInternalCorElementType();
```

Comment:
Change it here to `GetVerifierCorElementType` as well?

Reply:
Changed the interpreter too. Do we have any tests for that?